frontends/concrete-python/concrete/fhe/tfhers/dtypes.py

```diff
@@ -3,32 +3,142 @@
 """
 
 from functools import partial
-from typing import Any, Union
+from typing import Any, Optional, Union
 
 import numpy as np
 
 from ..dtypes import Integer
 
 
+class TFHERSParams:
```

Comment:
Maybe here we should have two part of parameters, one is for crypto parameters needed for changing partition (lwe_dim,glwe_dim,poly_size, ciphertext_modulus, noise_level) the other one is for the radix encoding (message_modulus, carry_modulus).
That will allow to separate the change_partition and the trans encoding between radix and native encoding, it will allow also the composability with other things than tfhe-rs.

Reply:
we could also maybe remove unnecessary parameters